demisto_sdk/commands/lint/helpers.py

```diff
@@ -257,58 +259,64 @@ def get_python_version_from_image(image: str) -> float:
         float: Python version X.Y (3.7, 3.6, ..)
     """
     docker_client = docker.from_env()
-    container_obj: Container = None
-    py_num = ""
-    for trial1 in range(2):
+    containers = list()
+    py_num = 2.7
+    # Run two times
+    for _ in range(2):
         try:
             command = "python -c \"import sys; print('{}.{}'.format(sys.version_info[0], sys.version_info[1]))\""
 
-            container_obj: Container = docker_client.containers.run(image=image,
-                                                                    command=shlex.split(command),
-                                                                    detach=True)
+            container_obj: Container = docker_client.containers.run(
+                image=image,
+                command=shlex.split(command),
+                detach=True
+            )
+            containers.append(container_obj)
             # Wait for container to finish
             container_obj.wait(condition="exited")
             # Get python version
             py_num = container_obj.logs()
             if isinstance(py_num, bytes):
                 py_num = float(py_num)
-                break
             else:
                 raise docker.errors.ContainerError
+            for _ in range(2):
+                # Try to remove the container two times.
+                try:
+                    container_obj.remove(force=True)
+                    break
+                except docker.errors.APIError:
+                    pass
         except (docker.errors.APIError, docker.errors.ContainerError):
             continue
 
-    if container_obj:
-        for trial2 in range(2):
-            try:
-                container_obj.remove(force=True)
-                break
-            except docker.errors.APIError:
-                continue
-
     return py_num
 
 
-def get_file_from_container(container_obj: Container, container_path: str, encoding: str = "") -> str:
+def get_file_from_container(container_obj: Container, container_path: str, encoding: str = "") -> Union[str, bytes]:
     """ Copy file from container.
 
     Args:
         container_obj(Container): Container ID to copy file from
         container_path(Path): Path in container image (file)
-        encoding(str): valide encoding e.g. utf-8
+        encoding(str): valid encoding e.g. utf-8
 
     Returns:
         str: file as string decode as utf-8
 
     Raises:
-        IOError: Rase IO error if unable to create temp file
+        IOError: Raise IO error if unable to create temp file
     """
+    data: Union[str, bytes] = b''
     archive, stat = container_obj.get_archive(container_path)
-    filelike = io.BytesIO(b"".join(b for b in archive))
-    tar = tarfile.open(fileobj=filelike)
-    data = tar.extractfile(stat['name']).read()
-    if encoding:
-        data = data.decode(encoding)
+    file_like = io.BytesIO(b"".join(b for b in archive))
+    tar: Optional[tarfile.TarFile] = tarfile.open(fileobj=file_like)
```

Comment:
```suggestion
    tar: tarfile.TarFile = tarfile.open(fileobj=file_like)
```

Reply:
it's Optional, mypy will fail without the optional annotation